lib/Controller/NavigationController.php

```diff
@@ -189,15 +189,15 @@ public function navigate(string $path = ''): TemplateResponse {
 	}
 
 	private function setupCloudAddress(): string {
-		$frontControllerActive =
-			($this->config->getSystemValue('htaccess.IgnoreFrontController', false) === true
-			 || getenv('front_controller_active') === 'true');
+//		$frontControllerActive =
+//			($this->config->getSystemValue('htaccess.IgnoreFrontController', false) === true
+//			 || getenv('front_controller_active') === 'true');
 
 		$cloudAddress = rtrim($this->config->getSystemValue('overwrite.cli.url', ''), '/');
 		if ($cloudAddress !== '') {
-			if (!$frontControllerActive) {
-				$cloudAddress .= '/index.php';
-			}
+//			if (!$frontControllerActive) {
```

Comment:
Why is this commented out? This will not work on instances where the rewrite is not set.